lib/ansible/modules/database/postgresql/postgresql_facts.py

```diff
@@ -0,0 +1,921 @@
+          returned: always
+          type: str
+          sample: postgres
+        size:
+          description: Database size in bytes.
+          returned: always
+          type: str
+          sample: 8189415
+extensions:
+  description:
+  - Extensions U(https://www.postgresql.org/docs/current/sql-createextension.html).
+  returned: always
+  type: dict
+  sample:
+  - { "plpgsql": { "description": "PL/pgSQL procedural language",
+    "extversion": { "major": 1, "minor": 0 } } }
+  contains:
+    extdescription:
+      description: Extension description.
+      returned: if existent
+      type: str
+      sample: PL/pgSQL procedural language
+    extversion:
+      description: Extension description.
+      returned: always
+      type: dict
+      contains:
+        major:
+          description: Extension major version.
+          returned: always
+          type: int
+          sample: 1
+        minor:
+          description: Extension minor version.
+          returned: always
+          type: int
+          sample: 0
+    nspname:
+      description: Namecpase where the extension is.
+      returned: always
+      type: str
+      sample: pg_catalog
+languages:
+  description: Procedural languages U(https://www.postgresql.org/docs/current/xplang.html).
+  returned: always
+  type: dict
+  sample: { "sql": { "lanacl": "", "lanowner": "postgres" } }
+  contains:
+    lanacl:
+      description:
+      - Language access privileges
+        U(https://www.postgresql.org/docs/current/catalog-pg-language.html).
+      returned: always
+      type: str
+      sample: "{postgres=UC/postgres,=U/postgres}"
+    lanowner:
+      description:
+      - Language owner U(https://www.postgresql.org/docs/current/catalog-pg-language.html).
+      returned: always
+      type: str
+      sample: postgres
+namespaces:
+  description:
+  - Namespaces (schema) U(https://www.postgresql.org/docs/current/sql-createschema.html).
+  returned: always
+  type: dict
+  sample: { "pg_catalog": { "nspacl": "{postgres=UC/postgres,=U/postgres}", "nspowner": "postgres" } }
+  contains:
+    nspacl:
+      description:
+      - Assess privileges U(https://www.postgresql.org/docs/current/catalog-pg-namespace.html).
+      returned: always
+      type: str
+      sample: "{postgres=UC/postgres,=U/postgres}"
+    nspowner:
+      description:
+      - Schema owner U(https://www.postgresql.org/docs/current/catalog-pg-namespace.html).
+      returned: always
+      type: str
+      sample: postgres
+repl_slots:
+  description:
+  - Replication slots (available in 9.4 and later)
+    U(https://www.postgresql.org/docs/current/catalog-pg-replication-slots.html).
+  returned: if existent
+  type: dict
+  sample: { "slot0": { "active": false, "database": null, "plugin": null, "slot_type": "physical" } }
+  contains:
+    active:
+      description: True if this slot is currently actively being used.
```

Comment:
I know this is the exact documentation from postgresl.org, but I think this 'sounds' as if a receiver is currently connected to it. Maybe we should add some info, as that 'True means that a receiver has connected to it, and it is currently reserving archives', or something like that.